crypto/crypto_aes.c

```diff
@@ -89,30 +91,32 @@ useaesni(void)
 #endif /* CPUSUPPORT_X86_AESNI */
 
 /**
- * crypto_aes_key_expand(key, len):
- * Expand the ${len}-byte AES key ${key} into a structure which can be passed
- * to crypto_aes_encrypt_block.  The length must be 16 or 32.
+ * crypto_aes_key_expand(key, keylen):
```

Comment:
I think we can normalize on `len` then; any time there's a `size_t len` parameter it should refer to the immediately preceding pointer value, so I don't think there's much risk of confusion.